src/routes/home/index.tsx

```diff
@@ -64,7 +65,14 @@ export default () => (
                     <li key={e.key}>
                       <a href={`/events/${e.key}`}>
                         {e.shortName}
-                        <DateDisplay date={e.parsedDate} />
+                        <div class={infoClass}>
+                          <p>
+                            {screen.width <= 768
```

Comment:
Nope. Use CSS media queries, not JS width checks. This does not work when you resize it

Reply:
How do you use media queries to abbreviate? I guess we could have two elems and only display one or another...

Reply:
Yeah, I think that is the best solution